### PR TITLE
feat(ui-components): new icon message sent added

### PR DESCRIPTION
**Pull request by sanat-thukral**
# Feature: new icon message send added

## Related Issue
#888 

## Description
- Adds a new icon message sent into ui-icons for consumption in Guided Answers

### CODE REVIEW

File: .changeset/brave-pumas-deny.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/ui-components': minor
```
Comment (815are) [inline .changeset/brave-pumas-deny.md:2]: I am not fully sure, but maybe it better fits to patch?
Like it is with other similar PRs - https://github.com/SAP/open-ux-tools/pull/879/files#diff-e5d3f21e663e649426773011139a16949b1247ba0335a4e28ad8246d2fc06128R2

Comment (sanat-thukral) [reply .changeset/brave-pumas-deny.md:2]: Thanks for pointing that out I'll change it now :)

**Review by eouin — DISMISSED** [review]

**Review by devinea — APPROVED** [review]

**Review by 815are — DISMISSED** [review]

**Review by Klaus-Keller — APPROVED** [review]
- code for adding the icon looks good
- did not built and test locally

Approved from my side.

**Review by 815are — APPROVED** [review]

**Comment by sanat-thukral** [comment]
Thanks @815are @devinea @eouin @Klaus-Keller for reviews 😄

**Pull request merged**